File: Python/1 - Iniciante/1046.py
# coding: UTF - 8
# Funcional no URI

A, B = input().split()
A = int(A)
B = int(B)
H = 0

if A >= 0 <= B <= 24 >= A:
    if A > B:
        H = int(24 - A + B)
    elif A < B:
        H = int(B - A)
    elif A == B:
        H = int(24)

print("O JOGO DUROU {:d} HORA(S)".format(H))

File: go/1046.go
// Funcional no beecrowd
package main

import (
	"fmt"
)

func main() {
	var A int
	var B int

	fmt.Scan(&A)
	fmt.Scan(&B)

	H := 0

	if A >= 0 && B >= 0 && A <= 24 && B <= 24 {
		if A > B {
			H = 24 - A + B
		} else if A < B {
			H = B - A
		} else if A == B {
			H = 24
		}
	}

	fmt.Println("O JOGO DUROU", H, "HORA(S)")
}
